mw-webapp/src/dataAccessLogic/DayReportDAL.ts

```diff
@@ -20,20 +21,27 @@ import {DateUtils} from "src/utils/DateUtils";
 export class DayReportDAL {
 
   /**
-   * Get DayReports
+   * Get DayReports that belong to a specific way
    */
-  public static async getDayReports(): Promise<DayReport[]> {
-    const dayReportsDTO = await DayReportService.getDayReportsDTO();
+  public static async getDayReports(wayUuid: string): Promise<DayReport[]> {
+    const dayReportsUuids = (await WayService.getWayDTO(wayUuid)).dayReportUuids;
+
+    const dayReportsDTO = await Promise.all(dayReportsUuids.map(async (uuid) => {
```

Comment:
I think no. Because we should get all day reports for specific way? that we do. And yes it includes too much reafs from firestore bit I don't know right now how to resolve it